src/Monitoring/LoopStatistics.cpp

```diff
@@ -88,28 +124,27 @@ void seissol::LoopStatistics::printSummary(MPI_Comm comm) {
     for (unsigned region = 0; region < nRegions; ++region) {
       double x  = sums[5*region + 0];
       double x2 = sums[5*region + 1];
-      double xy = sums[5*region + 2];
       double y  = sums[5*region + 3];
       double N  = sums[5*region + 4];
 
-      double det = N*x2 - x*x;
-      double regressionCoeffs[2];
-      regressionCoeffs[0] = (x2*y - x*xy) / det;
-      regressionCoeffs[1] = (-x*y + N*xy) / det;
-    
+      double xm = x / N;
+      double xv = x2 - 2*x*xm + xm*xm;
```

Comment:
Is there a reason why you've expanded the binomial?
Makes it imo a bit hard for "pattern matching" this with the formula

Reply:
Yes, because I may reuse the sums I already computed and sent around via MPI.